src/Person.src.json

```diff
@@ -126,7 +133,8 @@
         "input": {
             "@context": [
                 "http://schema4i.org/Thing.jsonld",
-                "http://schema4i.org/Person.jsonld"
+                "http://schema4i.org/Person.jsonld",
+                "http://schema4i.org/Occupation.jsonld"
             ],
```

Comment:
Please add `@context` declaration of `HasOccupation` with `@id` as `http://schema.org/hasOccupation` and `@type` as `http://schema4i.org/Occupation`. Please be aware of doing this in each instance if multiple types are possible for the same meaning.